speckle/Cache.py

```diff
@@ -3,11 +3,44 @@
 
 import os
 
+"""Speckle Cache documentation
+
+The SpeckleCache class is used to manage user profiles and cached 
+objects and streams.
+
+Example:
+    Instantiate a cache and retrieve existing user accounts::
+
+        from speckle import SpeckleCache
+
+        cache = SpeckleCache()
+
+        accounts = cache.get_all_accounts()
+
+        # Check if user profile exists
+        email = "foo@bar.com"
+        host = "foo.bar.com"
+        res = cache.account_exists(host, email)
+
+
+"""
 class SpeckleCache():
+    """Class for speckle cache.
 
+    """
     initialized = False
 
     def __init__(self, filepath=None):
+        """Initialize cache object
+
+        This creates a SpeckleCache object using either the default database 
+        location (%LOCALAPPDATA%/SpeckleSettings/SpeckleCache.db) or a user-
```

Comment:
Does this also work on Unix systems?

Reply:
Not sure... but needs to be wherever Speckle puts `SpeckleCache.db`.

I guess we should do an OS check for the default database path.

@didimitrie Any suggestions for where to stick the cache on Unix systems?

Reply:
I guess the question was also where the local app data variable is set? On unix you can usually dump it in something like `~/.speckle/`

Reply:
%LOCALAPPDATA% is a Windows variable. I'll change it to check the OS, and set it accordingly.